lib/runner.js

```diff
@@ -63,10 +77,12 @@ optionChain(chainableMethods, function (opts, title, fn) {
 }, Runner.prototype);
 
 Runner.prototype._addTestResult = function (result) {
-	if (result.result.metadata.type === 'test') {
+	var test = result.result;
+
+	if (test.metadata.type === 'test') {
 		this.stats.testCount++;
 
-		if (result.result.metadata.skipped) {
+		if (test.metadata.skipped && !test.metadata.todo) {
```

Comment:
Oh and here <https://github.com/sindresorhus/ava/blob/0e1886582833986b896c31859aa61d8da404e996/api.js#L207>.